src/services/schedule.ts

```diff
@@ -413,6 +413,13 @@ async function matchStrategyPrep(
   return { transaction, isStrategyTx, isLastPrepForNonce: strategyPrep.isLastForNonce };
 }
 
+function SendEmailForNonStrategyTx(scheduled: IScheduled) {
```

Comment:
that's much better (just `sendEmail` would've been enough) but you need to fix the capitalization